### PR TITLE
(Partner ID: cyware-238) README.md and increasing the auth token timeouts. Added No IOC Found message when the IOCs is not found in CTIX. Also added support to create Package in CTIX.

### PR DESCRIPTION
**Pull request by gladipero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
Hi @gladipero, Can you please fill up the contribution form - [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA)?

**Comment by gladipero** [comment]
Hey, @GuyAfik This is a copy of this PR(https://github.com/demisto/content/pull/15225), let me know if I have to fill the form again?

**Comment by GuyAfik** [comment]
> Hey, @GuyAfik This is a copy of this PR(#15225), let me know if I have to fill the form again?

Yes, please fill the form again, Thank you!

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 1]
```diff
@@ -352,30 +455,94 @@ def file_details_command(client: Client, args: Dict[str, Any]) -> List[CommandRe
...
+        "ips": args.get("ips", []),
+        "urls": args.get("urls", []),
+        "domains": args.get("domains", []),
+        "files": args.get("files", []),
+        "emails": args.get("emails", []),
+        "malwares": args.get("malwares", []),
+        "threat_actors": args.get("threat_actors", []),
+        "attack_patterns": args.get("attack_patterns", []),
+        "title": args.get("title", None),
+        "description": args.get("description", None),
```
Comment (GuyAfik) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:538]:
```suggestion
        "description": args.get("description"),
```

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 2]
```diff
@@ -352,30 +455,94 @@ def file_details_command(client: Client, args: Dict[str, Any]) -> List[CommandRe
...
+        "confidence": args.get("confidence", None),
```
Comment (GuyAfik) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:539]:
```suggestion
        "confidence": args.get("confidence"),
```

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 3]
```diff
@@ -352,30 +455,94 @@ def file_details_command(client: Client, args: Dict[str, Any]) -> List[CommandRe
...
+        "tlp": args.get("tlp", None),
```
Comment (GuyAfik) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:540]:
```suggestion
        "tlp": args.get("tlp"),
```

File: Packs/CTIX/ReleaseNotes/1_1_0.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Cyware Threat Intelligence eXchange
+- Added handling to the case where the input IOC is not found in CTIX.
+- Increased the timeout of token to 30 seconds instead of 5.
+- Updated the Docker image to: *demisto/python3:3.9.7.24076*.
```
Comment (GuyAfik) [inline Packs/CTIX/ReleaseNotes/1_1_0.md:6]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.1.25933*.
```

**Comment by GuyAfik** [comment]
Hi @gladipero, Thank you for your contribution, you have done an incredible job! a few points:
1) can you please add the new DBot Score calculation to ```CTIX_description.md```? The DBot score mapping calculation should be documented.
2) Can you please fix the build issues?
3) see my minor comments

**Comment by GuyAfik** [comment]
Hi @gladipero, can we schedule for a demo?

**Pull request merged**